kumascript/macros/LearnSidebar.ejs

```diff
@@ -1881,6 +1893,24 @@ var text = mdn.localStringMap({
             <li><a href="<%=baseURL%>Accessibility/Accessibility_troubleshooting"><%=text['Assessment_Accessibility_troubleshooting']%></a></li>
         </ol>
     </details>
+  </li>
+   <li><a href="<%=baseURL%>Web_Performance"><strong><%=text['Web_Performance_—_Making_websites_fast_and_responsive']%></a></strong></li>
+  <li class="toggle">
+    <details <%=currentPageIsUnder('Web_performance')%>>
+        <summary><%=text['Web_performance_guides']%></summary>
+        <ol>
+          <li><a href="<%=baseURL%>Web_performance"><%=text['The_why_of_web_performance']%></a></li>
```

Comment:
We need an item at the start called something like "Performance overview" linking to https://developer.mozilla.org/en-US/docs/Learn/Performance, because the title of this section (i.e. line 1900) isn't a link, so the overview page is inaccessible.

Reply:
Also, please check these links, they seem to be broken (the base URL is "/en-US/docs/Learn/Performance/").